database/postgres/postgres.go

```diff
@@ -133,9 +135,10 @@ func (p *Postgres) Open(url string) (database.Driver, error) {
 	}
 
 	px, err := WithInstance(db, &Config{
-		DatabaseName:     purl.Path,
-		MigrationsTable:  migrationsTable,
-		StatementTimeout: time.Duration(statementTimeout) * time.Millisecond,
+		DatabaseName:             purl.Path,
+		MigrationsTableHasSchema: ((migrationsTableHasSchemaString == "1") || (migrationsTableHasSchemaString == "true")),
```

Comment:
Use [strconv.ParseBool](https://golang.org/pkg/strconv/#ParseBool). Only parse if the string is not empty and errors should be handled.

Reply:
Done.